sig-architecture/annual-report-2024.md

```diff
@@ -11,16 +11,28 @@
    - Paying down significant tech debt
    - Governance and leadership changes
 -->
+- At Kubecon NA, we merged the last outstanding conformance test to reach "100% conformance tested" milestone (see [issue](https://github.com/kubernetes/enhancements/issues/4945)). With this we fully paid down the debt for older APIs which did not have conformance tests.
```

Comment:
What was that last one PR? The issue doesn't say. 

The link to the issue should be fine, but then I would expect that information there.

Reply:
![image](https://github.com/user-attachments/assets/6bb90f19-bab5-4392-99e1-45e55430ba06)

Reply:
I found it in parallel by going through the git history. That usage of "Discussion Link" is sneaky. I left a comment in the issue.